triton-inference-client.spec

```diff
@@ -44,25 +47,46 @@ cmake ${PROJ_DIR} \
     -DCMAKE_CXX_STANDARD=%{cms_cxx_standard} \
     -DTRITON_ENABLE_CC_HTTP=OFF \
     -DTRITON_ENABLE_CC_GRPC=ON \
-    -DTRITON_ENABLE_PYTHON_HTTP=OFF \
```

Comment:
If this part is only for building the C++ client (since there's a separate `buildpy` below), shouldn't these lines be kept here? (the defaults for all of these are OFF, right now, but I think it's better to be explicit, in case the default changes...)

Reply:
These `TRITON_ENABLE_PYTHON_*`  flags are not available for `triton/src/c++/CMakeLists.txt` ( which we use to build the c++ interface)  and `cmake` generate warning message that these are unused flags.

Reply:
Ah, that is because they are in the top-level CMakeLists.txt, but we're using the project-specific ones for more control. Okay, never mind then.